Cloud Application Manager/Core Concepts/context-switcher.md

```diff
@@ -0,0 +1,119 @@
+
+It provides at a glance all the necessary information so that users know in which scope they are situated at every moment. Thus, this component offers the following data when collapsed:
+
+* Kind of Scope (in blue)
+
+* Scope’s icon/avatar (in red)
+
+* Scope’s hierarchy (in green)
+
+![Context Switcher - In Detail](../../images/cloud-application-manager/context-switcher-detail.png)
+
+### Sections
+
+Clicking on the context switcher will create an overlay with extended information and functionality, and divided into different sections, which correspond to scope types (Organizations, Cost Centers and Workspaces), apart from a Favorites section:
+
+![Context Switcher - Main View](../../images/cloud-application-manager/context-switcher-general.png)
+
+#### Search bar
+
+This will let users search for a concrete scope they have access to. It is important to highlight that results will be different depending on which kind of scope the user has selected, so it won’t be possible to seek for a scope not belonging to the active section so, for example, when the Organization section is selected, only organizations are searched.
+
+![Context Switcher - Search Bar](../../images/cloud-application-manager/context-switcher-search-bar.png)
+
+#### Selection menu
+
+There are several selectable options here depending on user's permissions (organization administrators can see Organizations and Cost Centers options while cost center administrators could only see their associated Cost Centers), and apart from the different kinds of scopes (Organizations, Cost Centers and Workspaces), there is also a Favorites option, which allows to store together those scopes marked as favorites by the user. 
+
+![Context Switcher - Selection Menu](../../images/cloud-application-manager/context-switcher-selection-menu.png)
+
+#### Main view
+
+When an option is selected, main view displays all available scopes of selected kind for a given user, and depending on that kind of scope some differences arise:
+
+***Organizations view***: displays all the organizations the user has access to, and allows him or her to switch between them.
+
+![Context Switcher - Organizations View](../../images/cloud-application-manager/context-switcher-org-view.png)
+
+Every organization, as seen in the previous image, has several elements: name and icon of the organization, a star to mark it (or unmark if already marked) as a favorite, and a pencil (when hovered) to edit its details.
+
+![Context Switcher - Organizations Detail](../../images/cloud-application-manager/context-switcher-org-detail.png)
+
+***Cost Centers view***: this view introduces a left side panel where all the organizations the user has access to are listed, so it can be used as a filter. By default, the *All* option is selected, so every cost center depending on that organizations will be displayed in the main view. Selecting another organization, therefore, will show only its associated cost centers.
```

Comment:
```suggestion
##### Cost Centers view
This view introduces a left side panel where all the organizations the user has access to are listed, so it can be used as a filter. By default, the *All* option is selected, so every cost center depending on that organizations will be displayed in the main view. Selecting another organization, therefore, will show only its associated cost centers.
```